Tip for local backup CR name (#188)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/backup-restore.adoc

include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
----

> <1> The name of the Hazelcast custom resource for both backup and restore must be the same. If the Hazelcast custom resource name is different, the restore fails.

Reviewer: ```suggestion
<1> Hazelcast custom resource name for both backup and restore CRs must be the same, otherwise, the restore fails.
```

> <2> `HotBackup` resource name used for restore. The backup folder name will be taken from the `HotBackup` resource.

Reviewer: ```suggestion
<2> `HotBackup` resource name used for the restore. The backup folder name will be the name you provide here.
```

<3> Agent responsible for restoring data from the local storage.
The agent configuration is optional. If you give `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
uses the latest agent version that is compatible with its version.

> WARNING: You can only use a local backup once to restore a cluster. It is recommended to backup externally if you need to persistently restore a backup across clusters.

Reviewer: ```suggestion
WARNING: You can use a local backup only once to restore a cluster. We recommend you backup externally if you need to persistently restore a backup across the clusters.
```

> WARNING:: You can only use a local backup once to restore a cluster. It is recommended to backup externally if you need to persistently restore a backup across clusters.

Reviewer 2: ```suggestion
WARNING: You can only use a local backup once to restore a cluster. It is recommended to backup externally if you need to persistently restore a backup across clusters.
```
